src/main/menu.ts

```diff
@@ -76,6 +76,15 @@ function getHelpItems(): Array<MenuItemConstructorOptions> {
         shell.openExternal('https://github.com/electron/electron/issues');
       },
     },
+    {
+      type: 'separator',
+    },
+    {
+      label: 'About Electron Fiddle',
```

Comment:
Can we remove this menuItem for macOS? There's already the `About Electron Fiddle` menu item under the first submenu set by the `electron-default-menu` package: 
![image](https://user-images.githubusercontent.com/16010076/113763391-809af380-96ce-11eb-93b4-f918ca676ed2.png)

This positioning aligns with the behaviour of native macOS apps:
![image](https://user-images.githubusercontent.com/16010076/113763638-c2c43500-96ce-11eb-9cb6-bd4b942e4dbd.png)

Reply:
50231a12263a025b503aab357effbbc405fa53d0 omits on macOS now